modules/social_features/social_landing_page/social_landing_page.install

```diff
@@ -731,3 +730,12 @@ function social_landing_page_update_10302() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Revoke permission.
+ */
+function social_landing_page_update_10303(): void {
```

Comment:
@dmytriikaiun I don't think we need to add `:void` here and in other functions in this PR, since I think we need first to refactor all other code of the module to be consistent.